File: despatch.WEB/app/src/swagger/models/user-urls-response.ts
/* tslint:disable */
import { BaseEntityResponse } from './base-entity-response';
import { UrlResponse } from './url-response';
import { UserResponse } from './user-response';
export interface UserUrlsResponse extends BaseEntityResponse{
  url?: UrlResponse;
  urlId?: string;
  user?: UserResponse;
  userId?: string;
}

File: despatch.Domain/Models/Default/Response/UserUrlsResponse.cs
namespace despatch.Domain.Models.Default.Response
{
    public class UserUrlsResponse : BaseEntityResponse
    {
        public string UserId { get; set; }
        public string UrlId { get; set; }

        public virtual UserResponse User { get; set; }
        public virtual UrlResponse Url { get; set; }
    }
}
